siliconcompiler/tools/openroad/scripts/sc_procs.tcl

```diff
@@ -19,8 +19,10 @@ proc sc_global_placement_density {} {
   # User specified adjustment
   if { $openroad_gpl_uniform_placement_adjustment > 0.0 } {
     set or_uniform_adjusted_density \
-      [expr { $or_uniform_density + ((1.0 - $or_uniform_density) * \
-              $openroad_gpl_uniform_placement_adjustment) + $or_adjust_density_adder }]
+      [expr {
+        $or_uniform_density + ((1.0 - $or_uniform_density) *
+          $openroad_gpl_uniform_placement_adjustment) + $or_adjust_density_adder
```

Comment:
why is this indenting different? it seems like these two lines should have been aligned

Reply:
That's cause there's a parenthesized expression spanning both lines. The formatter does that to try and make parenthesized groupings clear, but if that's more distracting than helpful I can reconsider.

Reply:
I see that now.